sdk/src/wallet/storage/constants.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+// TODO: can we remove the slight inconsistencies in this file?
```

Comment:
Can you expand so we can answer this question?

Reply:
just the names seemed a bit inconsistent. `default_storage_path` is either returning something that is called a folder name or called a path. Should return relative paths in any case I think.

Reply:
I can remove the TODO tbh

Reply:
As you want but either address it or create an issue